docs/guides/using-org-slugs.mdx

```diff
@@ -0,0 +1,355 @@
+  | - | - | - |
+  | `/orgs/acmecorp` | Organization Acmecorp | Acmecorp's home |
+  | `/orgs/acmecorp/settings` | Organization Acmecorp | Acmecorp's settings |
+  | `/me` | Personal account | Personal home |
+  | `/me/settings` | Personal account | Personal settings |
+
+  ### Configuring `<OrganizationSwitcher />`
+
+  The [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) and
+  [`<OrganizationList />`](/docs/components/organization/organization-list) components each have a powerful set of options for
+  working with slugs and IDs.
+
+  This sample application uses the [personal account](/docs/organizations/organization-workspaces#organization-workspaces-in-the-clerk-dashboard:~:text=Personal%20account), so you can set `hidePersonal` to `false` to ensure the personal
+  account is selectable. You can also allow users to customize their organization's URL slug when they
+  create their organization initially by setting `hideSlug` to `false`.
+
+  Finally, you can give the components navigation instructions. If the user selects or creates an organization
+  with the slug `acmecorp`, by setting `afterCreateOrganizationUrl` and `afterSelectOrganizationUrl` to `'/orgs/:slug'`,
+  the component will navigate the user to `/orgs/acmecorp`. If the user selects their [personal account](/docs/organizations/organization-workspaces#organization-workspaces-in-the-clerk-dashboard:~:text=Personal%20account),
+  they will be navigated to `/me` via `afterSelectPersonalUrl`.
+
+  <Tabs items={["<OrganizationSwitcher />", "<OrganizationList />"]}>
+    <Tab>
+      ```tsx {{ filename: 'app/sidebar.tsx', mark: [6] }}
+      import { OrganizationSwitcher } from '@clerk/nextjs'
+
+      export default function Sidebar() {
+        return (
+          <OrganizationSwitcher
+            // prettier-ignore
+            hidePersonal={false}
+            hideSlug={false}
+            afterCreateOrganizationUrl="/orgs/:slug"
+            afterSelectOrganizationUrl="/orgs/:slug"
+            afterSelectPersonalUrl="/me"
+          />
+        )
+      }
+      ```
+    </Tab>
+
+    <Tab>
+      ```tsx {{ filename: 'app/organization-list/[[...organization-list]]/page.tsx', mark: [6] }}
+      import { OrganizationList } from '@clerk/nextjs'
+
+      export default function OrganizationListPage() {
+        return (
+          <OrganizationList
+            // prettier-ignore
+            hidePersonal={false}
+            hideSlug={false}
+            afterCreateOrganizationUrl="/orgs/:slug"
+            afterSelectOrganizationUrl="/orgs/:slug"
+            afterSelectPersonalUrl="/me"
+          />
+        )
+      }
+      ```
+    </Tab>
+  </Tabs>
+
+  > [!TIP]
+  > If you prefer to have an [organization ID](/docs/references/javascript/organization/organization#properties) appear in
+  > your URL rather than a slug, you can use `:id` instead of `:slug` with both `afterCreateOrganizationUrl` and
+  > `afterSelectOrganizationUrl`.
```

Comment:
It's more like "if your app uses organization IDs instead of slugs in URLs, you'll need to use :id instead if :slug to for navigation to work". I'll workshop another less-personal version.

Reply:
how about 9b2d01ed9ecd751f8caaeecf30270396f779cc5a
?